---
layout: page
title:  "STHELAR, a multi-tissue dataset linking spatial transcriptomics and histology for cell type annotation"
date:   2025-07-11 00:00:00 +0100
categories: [Spatial Transcriptomics]
tags: paper
image: /assets/img/sthelar_teaser.png
short_desc: A multi-tissue dataset linking spatial transcriptomics and histology for cell type annotation
published_at: Preprint BioRxiv 2025
---

Félicie Giraud-Sauveur, Quentin Blampey, Hakim Benkirane, Arianna Marinello, Paul-Henry Cournède, Stergios Christodoulidis

*Preprint BioRxiv 2025*

[[preprint](https://www.biorxiv.org/content/10.1101/2025.07.11.664123v1){:target="_blank"}] [[code](https://github.com/MICS-Lab/STHELAR){:target="_blank"}] [[doi](https://zenodo.org/records/15849373){:target="_blank"}]

<div class="row">
    <div class="mx-auto w-75 pb-5">
        <img src="/assets/img/sthelar_overview.jpg" alt="sthelar" width="100%"/>
    </div>
</div>

**Abstract:** Understanding the composition of the tumor microenvironment is critical for cancer research. Spatial transcriptomics profile gene expressions in spatial context, revealing tissue architecture and cellular heterogeneity, but its cost and technical complexity limit adoption. To address this issue, we introduce a pipeline to build STHELAR, a large-scale dataset that integrates spatial transcriptomics with Hematoxylin and Eosin (H&E) whole slide images for cell type annotation. The dataset comprises 31 human Xenium FFPE sections across 16 tissue types, for 22 cancerous and 9 non-cancerous patients. It contains over 11 million cells, each assigned to one of ten curated cell-type categories designed to accommodate a pan-cancer setting. Annotations were derived through Tangram-based alignment to single-cell reference atlases, followed by slide-specific clustering and differential expression analysis. Co-registered H&E images enabled extraction of over 500,000 patches with segmentation and classification masks. Quality control steps assessed segmentation accuracy, filtered out low-confidence regions, and verified annotation integrity. STHELAR provides a reference resource for developing models to predict cell-type annotations directly from histological images.

<pre>
@article{giraud2025sthelar,
  title={STHELAR, a multi-tissue dataset linking spatial transcriptomics and histology for cell type annotation},
  author={Giraud-Sauveur, F{\'e}licie and Blampey, Quentin and Benkirane, Hakim and Marinello, Arianna and Courn{\`e}de, Paul-Henry and Christodoulidis, Stergios},
  journal={bioRxiv},
  pages={2025--07},
  year={2025},
  publisher={Cold Spring Harbor Laboratory}
}
</pre>